io-engine/src/grpc/v1/snapshot.rs

```diff
@@ -557,7 +556,7 @@ impl SnapshotRpc for SnapshotService {
                         Some(destroy_snapshot_request::Pool::PoolName(name)) => {
                             Lvs::lookup(name)
                                 .ok_or(LvsError::RepDestroy {
-                                    source: Errno::ENOMEDIUM,
+                                    source: BsError::LvsNotFound {},
```

Comment:
I suspect this might not be correct as both ENOMEDIUM and ENOENT should not both map to the same error type.
ENOMEDIUM should map to PreCondFailed and ENOENT to NotFound IIRC

Reply:
There are preserved to be different:
ENOMEDIUM is used for "LVS aka Pool not found" condition: `BsError::LvsNotFound`.
ENOENT is used for "Volume not found": `BsError::VolNotFound`.